pkg/apis/mattermost/v1alpha1/clusterinstallation_utils.go

```diff
@@ -269,6 +269,17 @@ func (mattermost *ClusterInstallation) GenerateIngress(name, ingressName string,
 			},
 		},
 	}
+
+	if useTLS {
+		ingress.Spec.TLS = []v1beta1.IngressTLS{
+			{
+				Hosts:      []string{ingressName},
+				SecretName: strings.ReplaceAll(ingressName, ".", "-") + "-tls-cert",
```

Comment:
Can you explain what this line is doing? Does this mean that if you want to use TLS you need a secret named `INGRESS_NAME-tls-cert` for each ingress?

If so, I think this should be documented.

Reply:
yes, this secret the `cert-manager` will populate with the certificate that will be generated. this is a standard when using TLS.
you can also create this secret manually and add the certificates in the secret and when creating the ingress you say where are your secrets.

I will document, this and other things in a followup, if is that ok for you
@gabrieljackson

Reply:
@gabrieljackson PTAL

Reply:
Sounds good on a document follow-up!